src/components/NavBar/index.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+
+import { Layout, Input, Space, Dropdown, Menu, Avatar, Button, Form } from 'antd';
+
+import { SearchOutlined } from '@ant-design/icons';
+
+import { useAuth } from 'hooks/useAuth';
+

```

Comment:
lets remove these extra new lines. but group it like this

react
//space
3rd party imports
//space
in project absolute imports
//space
in project relative imports